libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/component/camera/CameraComponent.kt

```diff
@@ -111,6 +114,11 @@ internal class CameraComponent constructor(
                 }
             }
         }
+        coroutineScope.launch {
+            locationViewModel.state.filterNotNull().collect {
+                cameraViewModel.invoke(CameraAction.UpdateLocation(it))
```

Comment:
```suggestion
                updateCamera(it)
```
We can directly update the camera to transition to current location for the first time when location is initialized. We don't need to store any current location in `CameraState`

Reply:
As per internal discussion, this will be fixed in Camera when we start fixing it.